gradle.groovy script errors if mainapplication.java doesn't have package line right at the top

```
FAILURE: Build failed with an exception.

* Where:
Script '...\node_modules\react-native-unimodules\gradle.groovy' line: 38

* What went wrong:
A problem occurred evaluating project ':app'.
> Java or Kotlin file ...\MainApplication.java does not include package declaration
```

The issue goes away if I move the `package com.company.name;` line to above the comments:

errors:
```
/*
 * main application comment
 */
package com.company.name;
```

works:
```
package com.company.name;
/*
 * main application comment
 */
```

--- a/gradle.groovy
+++ b/gradle.groovy
@@ -26,14 +26,14 @@ def readPackageFromJavaOrKotlinFile(String filePath) {
   def file = new File(filePath)
   def fileReader = new BufferedReader(new FileReader(file))
   def fileContent = ""
-  while (fileContent.trim() == "") {
-    fileContent = fileReader.readLine()
+  while ((fileContent = fileReader.readLine()) != null) {
+    def match = fileContent =~ /^package ([0-9a-zA-Z._]*);?$/
+    if (match.size() == 1 && match[0].size() == 2) {
+      fileReader.close()
+      return match[0][1]
+    }
   }
   fileReader.close()
-  def match = fileContent =~ /^package ([0-9a-zA-Z._]*);?$/
-  if (match.size() == 1 && match[0].size() == 2) {
-    return match[0][1]
-  }
 
   throw new GradleException("Java or Kotlin file $file does not include package declaration")
 }

